### ISSUE TITLE
Delete parts of the output mosaic option

### ISSUE THREAD

**Issue by Scartography** [ungarj/mapchete#303]
* This should be easy for local filesystem
* S3 might be a bit trickier but there is a way to delete up to 1000 objects with one command
* Simple example of gathering keys and executing the deletion:

```
import boto3
from botocore.client import Config

timeout = 60
region = "eu-central-1"
bucket = "xxx"
key_prefix = "rgbnir/"

# for in_file in filelist:
s3_client = boto3.client(
    's3',
    config=Config(
        connect_timeout=timeout,
        read_timeout=timeout
    ),
    region_name=region
)

counter = 0
batches = {}
batch_items = []
batch_counter = 1

# Source of tiles to be deleted, create batches of 1000 paths
with open(in_file, "r") as in_list:
    for line in in_list:
        if "tif" in line:
            key = line.replace('"', '')
            key = key.replace('\n', '')
            key = key_prefix + key
            key = {"Key": key}
            batch_items.append(key)
            counter += 1
            if counter == 999:
                batches[batch_counter] = batch_items
                counter = 0
                batch_counter += 1
                batch_items = []

# delete via batches
for i, batch in batches.items():
    s3_client.delete_objects(
        Bucket=bucket,
        Delete={
         'Objects': batch,
         'Quiet': True
        },
        RequestPayer='requester',
    )
```

**Comment by ungarj** [comment]
Thanks, I will work on a command line tool `mapchete rm`

**Comment by ungarj** [comment]
closed by #306
